pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageBrokerRequestHandler.java

```diff
@@ -151,6 +152,7 @@ public void shutDown() {
   protected BrokerResponse handleRequest(long requestId, String query, SqlNodeAndOptions sqlNodeAndOptions,
       JsonNode request, @Nullable RequesterIdentity requesterIdentity, RequestContext requestContext,
       HttpHeaders httpHeaders, AccessControl accessControl) {
+    QueryThreadContext.setQueryEngine("mse");
```

Comment:
One decision I had to make is whether this should be an enum instead, but decided to use Strings to make it easier to add extensible engines in the future